dts/msm8226/msm8926-lg-madai.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <200 0>, <224 0>, <200 0x10001>, <224 0x10001>;
```

Comment:
You should only have one entry here, probably `<200 0>` for MSM8926.

Reply:
The vendor kernel's msm-id specifies all four, and the vendor fastboot log says `200/150/0x10001/0` for msm-id and board-id combination, but lk2nd loads using msm-id `<200, 0>`. So I am changing it.